CLI can't find metadata.json file unless previously created by GUI

If GUI has not previously created the ```ODK Briefcase Storage``` Directory and CLI is used to pull, briefcase creates the new folder, giving message

>The directory ./foo/ODK Briefcase Storage doesn't exist. Creating it

But later sends error such as

>Error pulling a form: java.nio.file.NoSuchFileException: ./foo/ODK Briefcase Storage/./foo/ODK Briefcase Storage/forms/foo/metadata.json (see the logs for more info)

In logs this kind of thing

> 2019-10-15 10:42:55,816 [ForkJoinPool-1-worker-9] ERROR o.o.b.o.PullFormFromAggregate - Error pulling a form
java.io.UncheckedIOException: java.nio.file.NoSuchFileException: ./foo/ODK Briefcase Storage/./foo/ODK Briefcase Storage/forms/formid/metadata.json
	at org.opendatakit.briefcase.reused.UncheckedFiles.write(UncheckedFiles.java:80)
	at org.opendatakit.briefcase.reused.UncheckedFiles.write(UncheckedFiles.java:73)
	at org.opendatakit.briefcase.model.form.FileSystemFormMetadataAdapter.serialize(FileSystemFormMetadataAdapter.java:123)
	at org.opendatakit.briefcase.model.form.FileSystemFormMetadataAdapter.persist(FileSystemFormMetadataAdapter.java:96)
	at org.opendatakit.briefcase.model.form.FormMetadataCommands.lambda$updateAsPulled$1(FormMetadataCommands.java:19)
	at org.opendatakit.briefcase.model.form.FileSystemFormMetadataAdapter.execute(FileSystemFormMetadataAdapter.java:43)
	at org.opendatakit.briefcase.pull.aggregate.PullFromAggregate.lambda$pull$11(PullFromAggregate.java:155)
	at org.opendatakit.briefcase.reused.job.Job.lambda$thenAccept$8(Job.java:134)
	at org.opendatakit.briefcase.reused.job.JobsRunner.lambda$launchAsync$1(JobsRunner.java:65)
	at java.base/java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1407)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1603)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
Caused by: java.nio.file.NoSuchFileException: ./foo/ODK Briefcase Storage/./foo/ODK Briefcase Storage/forms/formid/metadata.json
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116)
	at java.base/sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:215)
	at java.base/java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:434)
	at java.base/java.nio.file.Files.newOutputStream(Files.java:218)
	at java.base/java.nio.file.Files.write(Files.java:3355)
	at org.opendatakit.briefcase.reused.UncheckedFiles.write(UncheckedFiles.java:78)
	... 12 common frames omitted
2019-10-15 10:44:32,344 [main] ERROR o.o.briefcase.ui.MainBriefcaseWindow - Failed to launch GUI
java.io.UncheckedIOException: java.nio.file.NoSuchFileException: ./foo/ODK Briefcase Storage/./foo/ODK Briefcase Storage/forms/formid/metadata.json
	at org.opendatakit.briefcase.reused.UncheckedFiles.write(UncheckedFiles.java:80)
	at org.opendatakit.briefcase.reused.UncheckedFiles.write(UncheckedFiles.java:73)
	at org.opendatakit.briefcase.model.form.FileSystemFormMetadataAdapter.serialize(FileSystemFormMetadataAdapter.java:123)
	at org.opendatakit.briefcase.model.form.FileSystemFormMetadataAdapter.persist(FileSystemFormMetadataAdapter.java:96)
	at java.base/java.util.stream.ReferencePipeline$11$1.accept(ReferencePipeline.java:441)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1492)


Looks like CLI is trying to update the .json file used for tracking cursor positions, but doesn't know what to do if the file is not present.

Export still stalls, but this is data downloaded from an old version of Aggregate (v1.4.15 Production)

```
Exporting form MSCEPI_MAPPING_ODK (MSC_MAPPING_EPI) to: ./MSC_MAPPING_EPI3
2019-10-21 16:19:23,786 [ForkJoinPool-1-worker-9] ERROR o.o.briefcase.reused.job.JobsRunner - Error running Job
org.opendatakit.briefcase.model.CryptoException: org.opendatakit.briefcase.model.CryptoException: Can't decrypt file
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:488)
	at java.base/java.util.concurrent.ForkJoinTask.getThrowableException(ForkJoinTask.java:600)
	at java.base/java.util.concurrent.ForkJoinTask.reportException(ForkJoinTask.java:678)
	at java.base/java.util.concurrent.ForkJoinTask.invoke(ForkJoinTask.java:737)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateParallel(ForEachOps.java:159)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateParallel(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:233)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:575)
	at org.opendatakit.briefcase.export.ExportToCsv.export(ExportToCsv.java:111)
	at org.opendatakit.briefcase.export.ExportToCsv.export(ExportToCsv.java:56)
	at org.opendatakit.briefcase.operations.Export.lambda$export$7(Export.java:156)
	at org.opendatakit.briefcase.reused.job.Job.lambda$run$0(Job.java:54)
	at org.opendatakit.briefcase.reused.job.Job.lambda$thenRun$7(Job.java:123)
	at org.opendatakit.briefcase.reused.job.Job.lambda$thenRun$7(Job.java:122)
	at org.opendatakit.briefcase.reused.job.Job.lambda$thenRun$6(Job.java:109)
	at org.opendatakit.briefcase.reused.job.JobsRunner.lambda$launchAsync$1(JobsRunner.java:65)
	at java.base/java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1407)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1603)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
Caused by: org.opendatakit.briefcase.model.CryptoException: Can't decrypt file
	at org.opendatakit.briefcase.export.SubmissionParser.decryptFile(SubmissionParser.java:213)
	at org.opendatakit.briefcase.export.SubmissionParser.decrypt(SubmissionParser.java:192)
	at org.opendatakit.briefcase.export.SubmissionParser.lambda$parseSubmission$5(SubmissionParser.java:149)
	at java.base/java.util.Optional.flatMap(Optional.java:283)
	at org.opendatakit.briefcase.export.SubmissionParser.parseSubmission(SubmissionParser.java:129)
	at org.opendatakit.briefcase.export.ExportTools.lambda$getValidSubmissions$0(ExportTools.java:30)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1492)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:290)
	at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:746)
	... 3 common frames omitted
Caused by: java.io.IOException: javax.crypto.BadPaddingException: pad block corrupted
	at java.base/javax.crypto.CipherInputStream.getMoreData(CipherInputStream.java:121)
	at java.base/javax.crypto.CipherInputStream.read(CipherInputStream.java:235)
	at java.base/javax.crypto.CipherInputStream.read(CipherInputStream.java:212)
	at org.opendatakit.briefcase.export.SubmissionParser.decryptFile(SubmissionParser.java:208)
	... 13 common frames omitted
Caused by: javax.crypto.BadPaddingException: pad block corrupted
	at org.bouncycastle.jce.provider.JCEBlockCipher.engineDoFinal(Unknown Source)
	at java.base/javax.crypto.Cipher.doFinal(Cipher.java:2074)
	at java.base/javax.crypto.CipherInputStream.getMoreData(CipherInputStream.java:118)
	... 16 common frames omitted
Error exporting a form: org.opendatakit.briefcase.model.CryptoException: Can't decrypt file (see the logs for more info)
2019-10-21 16:19:23,794 [ForkJoinPool-1-worker-9] ERROR o.o.briefcase.operations.Export - Error exporting a form
org.opendatakit.briefcase.model.CryptoException: org.opendatakit.briefcase.model.CryptoException: Can't decrypt file
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:488)
	at java.base/java.util.concurrent.ForkJoinTask.getThrowableException(ForkJoinTask.java:600)
	at java.base/java.util.concurrent.ForkJoinTask.reportException(ForkJoinTask.java:678)
	at java.base/java.util.concurrent.ForkJoinTask.invoke(ForkJoinTask.java:737)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateParallel(ForEachOps.java:159)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateParallel(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:233)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:575)
	at org.opendatakit.briefcase.export.ExportToCsv.export(ExportToCsv.java:111)
	at org.opendatakit.briefcase.export.ExportToCsv.export(ExportToCsv.java:56)
	at org.opendatakit.briefcase.operations.Export.lambda$export$7(Export.java:156)
	at org.opendatakit.briefcase.reused.job.Job.lambda$run$0(Job.java:54)
	at org.opendatakit.briefcase.reused.job.Job.lambda$thenRun$7(Job.java:123)
	at org.opendatakit.briefcase.reused.job.Job.lambda$thenRun$7(Job.java:122)
	at org.opendatakit.briefcase.reused.job.Job.lambda$thenRun$6(Job.java:109)
	at org.opendatakit.briefcase.reused.job.JobsRunner.lambda$launchAsync$1(JobsRunner.java:65)
	at java.base/java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1407)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1603)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
Caused by: org.opendatakit.briefcase.model.CryptoException: Can't decrypt file
	at org.opendatakit.briefcase.export.SubmissionParser.decryptFile(SubmissionParser.java:213)
	at org.opendatakit.briefcase.export.SubmissionParser.decrypt(SubmissionParser.java:192)
	at org.opendatakit.briefcase.export.SubmissionParser.lambda$parseSubmission$5(SubmissionParser.java:149)
	at java.base/java.util.Optional.flatMap(Optional.java:283)
	at org.opendatakit.briefcase.export.SubmissionParser.parseSubmission(SubmissionParser.java:129)
	at org.opendatakit.briefcase.export.ExportTools.lambda$getValidSubmissions$0(ExportTools.java:30)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1492)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:290)
	at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:746)
	... 3 common frames omitted
Caused by: java.io.IOException: javax.crypto.BadPaddingException: pad block corrupted
	at java.base/javax.crypto.CipherInputStream.getMoreData(CipherInputStream.java:121)
	at java.base/javax.crypto.CipherInputStream.read(CipherInputStream.java:235)
	at java.base/javax.crypto.CipherInputStream.read(CipherInputStream.java:212)
	at org.opendatakit.briefcase.export.SubmissionParser.decryptFile(SubmissionParser.java:208)
	... 13 common frames omitted
Caused by: javax.crypto.BadPaddingException: pad block corrupted
	at org.bouncycastle.jce.provider.JCEBlockCipher.engineDoFinal(Unknown Source)
	at java.base/javax.crypto.Cipher.doFinal(Cipher.java:2074)
	at java.base/javax.crypto.CipherInputStream.getMoreData(CipherInputStream.java:118)
	... 16 common frames omitted

All operations completed
```

--- a/src/org/opendatakit/briefcase/operations/Common.java
+++ b/src/org/opendatakit/briefcase/operations/Common.java
@@ -25,18 +25,18 @@ import org.opendatakit.briefcase.reused.http.RequestBuilder;
 import org.opendatakit.common.cli.Param;
 
 public class Common {
-  static final Param<String> STORAGE_DIR = Param.arg("sd", "storage_directory", "Briefcase storage directory");
+  public static final Param<String> DEPRECATED_AGGREGATE_SERVER = Param.arg("url", "aggregate_url", "Aggregate server URL");
+  public static final Param<URL> SERVER_URL = Param.arg("U", "odk_url", "ODK Server URL", RequestBuilder::url);
+  public static final Param<Integer> MAX_HTTP_CONNECTIONS = Param.arg("mhc", "max_http_connections", "Maximum simultaneous HTTP connections (defaults to 8)", Integer::parseInt);
+  static final Param<Path> STORAGE_DIR = Param.arg("sd", "storage_directory", "Briefcase storage directory", Paths::get);
   static final Param<String> FORM_ID = Param.arg("id", "form_id", "Form ID");
   static final Param<Integer> PROJECT_ID = Param.arg("pid", "project_id", "ODK Project ID number", Integer::parseInt);
   static final Param<String> CREDENTIALS_USERNAME = Param.arg("u", "odk_username", "ODK Username");
   static final Param<String> CREDENTIALS_EMAIL = Param.arg("E", "odk_email", "ODK Email");
   static final Param<String> CREDENTIALS_PASSWORD = Param.arg("p", "odk_password", "ODK Password");
-  public static final Param<String> DEPRECATED_AGGREGATE_SERVER = Param.arg("url", "aggregate_url", "Aggregate server URL");
-  public static final Param<URL> SERVER_URL = Param.arg("U", "odk_url", "ODK Server URL", RequestBuilder::url);
-  public static final Param<Integer> MAX_HTTP_CONNECTIONS = Param.arg("mhc", "max_http_connections", "Maximum simultaneous HTTP connections (defaults to 8)", Integer::parseInt);
 
-  static Path getOrCreateBriefcaseDir(String storageDir) {
-    Path briefcaseDir = BriefcasePreferences.buildBriefcaseDir(Paths.get(storageDir));
+  static Path getOrCreateBriefcaseDir(Path storageDir) {
+    Path briefcaseDir = BriefcasePreferences.buildBriefcaseDir(storageDir);
     if (!Files.exists(briefcaseDir)) {
       System.err.println("The directory " + briefcaseDir.toString() + " doesn't exist. Creating it");
       UncheckedFiles.createBriefcaseDir(briefcaseDir);

--- a/src/org/opendatakit/briefcase/operations/Common.java
+++ b/src/org/opendatakit/briefcase/operations/Common.java
@@ -48,7 +48,7 @@ public class Common {
    * Returns an absolute path to the given string path, using the user.dir property
    * to resolve input relative paths if necessary
    */
-  private static Path absolutePath(String path) {
+  static Path absolutePath(String path) {
     Path storageDir = Paths.get(path);
     return storageDir.isAbsolute()
         ? storageDir

--- a/src/org/opendatakit/briefcase/operations/Common.java
+++ b/src/org/opendatakit/briefcase/operations/Common.java
@@ -28,12 +28,7 @@ public class Common {
   public static final Param<String> DEPRECATED_AGGREGATE_SERVER = Param.arg("url", "aggregate_url", "Aggregate server URL");
   public static final Param<URL> SERVER_URL = Param.arg("U", "odk_url", "ODK Server URL", RequestBuilder::url);
   public static final Param<Integer> MAX_HTTP_CONNECTIONS = Param.arg("mhc", "max_http_connections", "Maximum simultaneous HTTP connections (defaults to 8)", Integer::parseInt);
-  static final Param<Path> STORAGE_DIR = Param.arg("sd", "storage_directory", "Briefcase storage directory", path -> {
-    Path storageDir = Paths.get(path);
-    return storageDir.isAbsolute()
-        ? storageDir
-        : Paths.get(System.getProperty("user.dir")).resolve(storageDir);
-  });
+  static final Param<Path> STORAGE_DIR = Param.arg("sd", "storage_directory", "Briefcase storage directory", Common::absolutePath);
   static final Param<String> FORM_ID = Param.arg("id", "form_id", "Form ID");
   static final Param<Integer> PROJECT_ID = Param.arg("pid", "project_id", "ODK Project ID number", Integer::parseInt);
   static final Param<String> CREDENTIALS_USERNAME = Param.arg("u", "odk_username", "ODK Username");
@@ -47,5 +42,16 @@ public class Common {
       UncheckedFiles.createBriefcaseDir(briefcaseDir);
     }
     return briefcaseDir;
+  }
+
+  /**
+   * Returns an absolute path to the given string path, using the user.dir property
+   * to resolve input relative paths if necessary
+   */
+  private static Path absolutePath(String path) {
+    Path storageDir = Paths.get(path);
+    return storageDir.isAbsolute()
+        ? storageDir
+        : Paths.get(System.getProperty("user.dir")).resolve(storageDir);
   }
 }

--- a/src/org/opendatakit/briefcase/operations/Common.java
+++ b/src/org/opendatakit/briefcase/operations/Common.java
@@ -28,7 +28,12 @@ public class Common {
   public static final Param<String> DEPRECATED_AGGREGATE_SERVER = Param.arg("url", "aggregate_url", "Aggregate server URL");
   public static final Param<URL> SERVER_URL = Param.arg("U", "odk_url", "ODK Server URL", RequestBuilder::url);
   public static final Param<Integer> MAX_HTTP_CONNECTIONS = Param.arg("mhc", "max_http_connections", "Maximum simultaneous HTTP connections (defaults to 8)", Integer::parseInt);
-  static final Param<Path> STORAGE_DIR = Param.arg("sd", "storage_directory", "Briefcase storage directory", Paths::get);
+  static final Param<Path> STORAGE_DIR = Param.arg("sd", "storage_directory", "Briefcase storage directory", path -> {
+    Path storageDir = Paths.get(path);
+    return storageDir.isAbsolute()
+        ? storageDir
+        : Paths.get(System.getProperty("user.dir")).resolve(storageDir);
+  });
   static final Param<String> FORM_ID = Param.arg("id", "form_id", "Form ID");
   static final Param<Integer> PROJECT_ID = Param.arg("pid", "project_id", "ODK Project ID number", Integer::parseInt);
   static final Param<String> CREDENTIALS_USERNAME = Param.arg("u", "odk_username", "ODK Username");

--- a/src/org/opendatakit/briefcase/operations/Export.java
+++ b/src/org/opendatakit/briefcase/operations/Export.java
@@ -96,7 +96,7 @@ public class Export {
       Arrays.asList(PEM_FILE, EXCLUDE_MEDIA, OVERWRITE, START, END, PULL_BEFORE, SPLIT_SELECT_MULTIPLES, INCLUDE_GEOJSON_EXPORT, REMOVE_GROUP_NAMES, SMART_APPEND)
   );
 
-  public static void export(String storageDir, String formid, Path exportDir, String baseFilename, boolean exportMedia, boolean overwriteFiles, boolean pullBefore, Optional<LocalDate> startDate, Optional<LocalDate> endDate, Optional<Path> maybePemFile, boolean splitSelectMultiples, boolean includeGeoJsonExport, boolean removeGroupNames, boolean smartAppend) {
+  public static void export(Path storageDir, String formid, Path exportDir, String baseFilename, boolean exportMedia, boolean overwriteFiles, boolean pullBefore, Optional<LocalDate> startDate, Optional<LocalDate> endDate, Optional<Path> maybePemFile, boolean splitSelectMultiples, boolean includeGeoJsonExport, boolean removeGroupNames, boolean smartAppend) {
     CliEventsCompanion.attach(log);
     Path briefcaseDir = Common.getOrCreateBriefcaseDir(storageDir);
     FormCache formCache = FormCache.from(briefcaseDir);

--- a/src/org/opendatakit/briefcase/operations/Export.java
+++ b/src/org/opendatakit/briefcase/operations/Export.java
@@ -24,7 +24,6 @@ import static org.opendatakit.briefcase.reused.UncheckedFiles.createDirectories;
 import static org.opendatakit.briefcase.reused.http.Http.DEFAULT_HTTP_CONNECTIONS;
 
 import java.nio.file.Path;
-import java.nio.file.Paths;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.util.Arrays;
@@ -61,13 +60,13 @@ import org.slf4j.LoggerFactory;
 public class Export {
   private static final Logger log = LoggerFactory.getLogger(Export.class);
   private static final Param<Void> EXPORT = Param.flag("e", "export", "Export a form");
-  private static final Param<Path> EXPORT_DIR = Param.arg("ed", "export_directory", "Export directory", Paths::get);
+  private static final Param<Path> EXPORT_DIR = Param.arg("ed", "export_directory", "Export directory", Common::absolutePath);
   private static final Param<String> FILE = Param.arg("f", "export_filename", "Filename for export operation");
   private static final Param<LocalDate> START = Param.localDate("start", "export_start_date", "Export start date (inclusive)");
   private static final Param<LocalDate> END = Param.localDate("end", "export_end_date", "Export end date (inclusive)");
   private static final Param<Void> EXCLUDE_MEDIA = Param.flag("em", "exclude_media_export", "Exclude media in export");
   private static final Param<Void> OVERWRITE = Param.flag("oc", "overwrite_csv_export", "Overwrite files during export");
-  private static final Param<Path> PEM_FILE = Param.arg("pf", "pem_file", "PEM file for form decryption", Paths::get);
+  private static final Param<Path> PEM_FILE = Param.arg("pf", "pem_file", "PEM file for form decryption", Common::absolutePath);
   private static final Param<Void> PULL_BEFORE = Param.flag("pb", "pull_before", "Pull before export");
   private static final Param<Void> SPLIT_SELECT_MULTIPLES = Param.flag("ssm", "split_select_multiples", "Split select multiple fields");
   private static final Param<Void> INCLUDE_GEOJSON_EXPORT = Param.flag("ig", "include_geojson", "Include a GeoJSON file with spatial data");

--- a/src/org/opendatakit/briefcase/operations/ImportFromODK.java
+++ b/src/org/opendatakit/briefcase/operations/ImportFromODK.java
@@ -20,7 +20,6 @@ import static org.opendatakit.briefcase.operations.Common.FORM_ID;
 import static org.opendatakit.briefcase.operations.Common.STORAGE_DIR;
 
 import java.nio.file.Path;
-import java.nio.file.Paths;
 import java.util.Arrays;
 import java.util.Optional;
 import org.opendatakit.briefcase.model.FormStatus;
@@ -37,7 +36,7 @@ import org.slf4j.LoggerFactory;
 public class ImportFromODK {
   private static final Logger log = LoggerFactory.getLogger(ImportFromODK.class);
   private static final Param<Void> IMPORT = Param.flag("pc", "pull_collect", "Pull from Collect");
-  private static final Param<Path> ODK_DIR = Param.arg("od", "odk_directory", "ODK directory", Paths::get);
+  private static final Param<Path> ODK_DIR = Param.arg("od", "odk_directory", "ODK directory", Common::absolutePath);
 
   public static final Operation IMPORT_FROM_ODK = Operation.of(
       IMPORT,

--- a/src/org/opendatakit/briefcase/operations/ImportFromODK.java
+++ b/src/org/opendatakit/briefcase/operations/ImportFromODK.java
@@ -50,7 +50,7 @@ public class ImportFromODK {
       Arrays.asList(FORM_ID)
   );
 
-  public static void importODK(String storageDir, Path odkDir, Optional<String> formId) {
+  public static void importODK(Path storageDir, Path odkDir, Optional<String> formId) {
     CliEventsCompanion.attach(log);
     Path briefcaseDir = Common.getOrCreateBriefcaseDir(storageDir);
     FormCache formCache = FormCache.from(briefcaseDir);

--- a/src/org/opendatakit/briefcase/operations/PullFormFromAggregate.java
+++ b/src/org/opendatakit/briefcase/operations/PullFormFromAggregate.java
@@ -81,7 +81,7 @@ public class PullFormFromAggregate {
       Arrays.asList(RESUME_LAST_PULL, INCLUDE_INCOMPLETE, FORM_ID, START_FROM_DATE, MAX_HTTP_CONNECTIONS)
   );
 
-  public static void pullFormFromAggregate(String storageDir, Optional<String> formId, String username, String password, URL server, boolean resumeLastPull, Optional<LocalDate> startFromDate, boolean includeIncomplete, Optional<Integer> maybeMaxHttpConnections) {
+  public static void pullFormFromAggregate(Path storageDir, Optional<String> formId, String username, String password, URL server, boolean resumeLastPull, Optional<LocalDate> startFromDate, boolean includeIncomplete, Optional<Integer> maybeMaxHttpConnections) {
     CliEventsCompanion.attach(log);
     Path briefcaseDir = Common.getOrCreateBriefcaseDir(storageDir);
     FormCache formCache = FormCache.from(briefcaseDir);

--- a/src/org/opendatakit/briefcase/operations/PushFormToAggregate.java
+++ b/src/org/opendatakit/briefcase/operations/PushFormToAggregate.java
@@ -68,7 +68,7 @@ public class PushFormToAggregate {
       Arrays.asList(FORCE_SEND_BLANK, MAX_HTTP_CONNECTIONS, FORM_ID)
   );
 
-  private static void pushFormToAggregate(String storageDir, Optional<String> formid, String username, String password, URL server, boolean forceSendBlank, Optional<Integer> maybeMaxConnections) {
+  private static void pushFormToAggregate(Path storageDir, Optional<String> formid, String username, String password, URL server, boolean forceSendBlank, Optional<Integer> maybeMaxConnections) {
     CliEventsCompanion.attach(log);
     Path briefcaseDir = Common.getOrCreateBriefcaseDir(storageDir);
     FormCache formCache = FormCache.from(briefcaseDir);

--- a/src/org/opendatakit/briefcase/ui/BriefcaseCLI.java
+++ b/src/org/opendatakit/briefcase/ui/BriefcaseCLI.java
@@ -21,6 +21,7 @@ import static org.opendatakit.briefcase.operations.Export.export;
 import static org.opendatakit.briefcase.operations.ImportFromODK.importODK;
 import static org.opendatakit.briefcase.operations.PullFormFromAggregate.pullFormFromAggregate;
 
+import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.text.DateFormat;
 import java.text.SimpleDateFormat;
@@ -289,7 +290,7 @@ public class BriefcaseCLI {
     String password = cli.getOptionValue(ODK_PASSWORD);
     String server = cli.getOptionValue(AGGREGATE_URL);
     String formid = cli.getOptionValue(FORM_ID);
-    String storageDir = cli.getOptionValue(STORAGE_DIRECTORY);
+    Path storageDir = Paths.get(cli.getOptionValue(STORAGE_DIRECTORY));
     String fileName = cli.getOptionValue(EXPORT_FILENAME);
     String exportPath = cli.getOptionValue(EXPORT_DIRECTORY);
     String startDateString = cli.getOptionValue(EXPORT_START_DATE);

